Makefile

```diff
@@ -0,0 +1,7 @@
+IMAGE_NAME := koji-build-finder
+
+build:
+	docker build --tag=$(IMAGE_NAME) .
+
+shell:
+	docker run -v $(PWD):/src -w /src -it $(IMAGE_NAME) bash
```

Comment:
I had to use `docker run -v $(CURDIR):/src` as PWD doesn't not work for me. I think you could also use `$(shell pwd)`

Reply:
can do, probably a difference in make version